roles/libvirt_manager/tasks/start_manage_vms.yml

```diff
@@ -1,9 +1,13 @@
 ---
-- name: Refresh pools before starting VMs
+- name: Refresh oooq_pool before starting VMs
+  when:
+    - item.value.state == 'running'
```

Comment:
Based on the checks in the beaker node, this should be `active` not `running`.

Reply:
I seem to have mixed things up:
```YAML
        "default": {
            "autostart": "yes",
            "path": "/var/lib/libvirt/images",
            "persistent": "yes",
            "size_available": "497357316096",
            "size_total": "506051166208",
            "size_used": "8693850112",
            "state": "active",
            "status": "running",
            "type": "dir",
            "uuid": "c9eb02d6-c91b-4994-8761-669dd2c8a10d",
            "volume_count": 0,
            "volumes": []
        },
```

IMHO checking for `status: running` is better than `state: active`. Updated accordingly.